systemtests/src/main/java/io/enmasse/systemtest/apiclients/UserApiClient.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2017-2018, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.apiclients;
+
+import io.enmasse.systemtest.CustomLogger;
+import io.enmasse.systemtest.Kubernetes;
+import io.enmasse.systemtest.User;
+import io.enmasse.systemtest.UserCredentials;
+import io.enmasse.systemtest.timemeasuring.Operation;
+import io.enmasse.systemtest.timemeasuring.TimeMeasuringSystem;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.codec.BodyCodec;
+import org.slf4j.Logger;
+
+import static java.net.HttpURLConnection.HTTP_CREATED;
+import static java.net.HttpURLConnection.HTTP_OK;
+
+import javax.ws.rs.core.HttpHeaders;
+import java.net.MalformedURLException;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+public class UserApiClient extends ApiClient {
+
+    private static Logger log = CustomLogger.getLogger();
+    private final int initRetry = 10;
+    private final String userPathPattern;
+
+    public UserApiClient(Kubernetes kubernetes) throws MalformedURLException {
+        super(kubernetes, kubernetes.getRestEndpoint(), "user.enmasse.io/v1alpha1");
+        this.userPathPattern = "/apis/user.enmasse.io/v1alpha1/namespaces/%s/messagingusers";
+    }
+
+    public void createUser(String addressSpace, String namespace, UserCredentials credentials) throws Exception {
+        User user = new User()
+                .setUsername(credentials.getUsername())
+                .setPassword(credentials.getPassword())
+                .addAuthorization(new User.AuthorizationRule()
+                        .addAddress("*")
+                        .addOperation(User.Operation.SEND)
+                        .addOperation(User.Operation.RECEIVE)
+                        .addOperation(User.Operation.VIEW))
+                .addAuthorization(new User.AuthorizationRule()
+                        .addAddress("*")
```

Comment:
@kornys This was intentionally to be without addresses. Manage does not apply to individual addresses, and the api allows empty list of addresses to signal that kind of operation